src/app/body/filter-page/edit-filter/edit-filter.component.css

```diff
@@ -11,16 +11,109 @@
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
 * See the MIT License for more details. 
 ***********************************************************/
+#openEditPropertiesContainer,
+.modal-content{
+  background-color: var(--secondary-bg);
+  color: var(--button-text);
+}
+
+.btn {
+    color: var(--text);
+}
+
+
 #button {
     background-color: var(--primary-color);
     color: white;
 }
 
+.btn:hover,
 #button:focus,
 #button:active,
 #button:hover {
-    outline: none;
-    box-shadow: none;
+    background-color: var(--dropdown-hover);
+    color: var(--button-text-hover);
+    border: none;
+}
+
+#SuccessIcon{
+    color: var(--secondary-color);
+    font-size: 48px;
+
+}
+
+.clr {
+    border: none;
+    color: var(--text);
+    background-color: var(--primary-bg);
+}
+
+.card {
+    background-color: var(--secondary-bg);
+    color: var(--text);
+    width: 24rem
```

Comment:
Make the width responsive. Giving it in 'rem' will make it static and the card will not resize according to the screen or device.

Reply:
okay I will make the changes